components/handlers/module_view_in_ay_sem.py

```diff
@@ -119,12 +119,14 @@ def GET(self):
         is_future_ay = not self.is_current_ay
 
         overlapping_mod_list = model.get_mod_taken_together_with(module_code)
+        prereq_string = model.get_prerequisite_as_string(module_code)
+        preclude_string = model.get_preclusion_as_string(module_code)
         is_starred = model.is_module_starred(module_code, web.cookies().get('user'))
 
         return RENDER.individualModuleInfo(module_info, is_future_ay,
                                            target_ay_sem, self.mounting_status,
                                            self.quota, overlapping_mod_list,
-                                           is_starred)
+                                           is_starred, prereq_string, preclude_string)
```

Comment:
Does module view in ay sem need prereq and preclusion data? Currently the data are passed to the UI but the UI does not render or use the data in any way.